src/SfxWeb/src/app/Models/DataModels/collections/infrastructureCollection.ts

```diff
@@ -0,0 +1,106 @@
+import { forkJoin, Observable, of } from 'rxjs';
+import { map, mergeMap } from 'rxjs/operators';
+import { Constants, StatusWarningLevel } from 'src/app/Common/Constants';
+import { IResponseMessageHandler } from 'src/app/Common/ResponseMessageHandlers';
+import { DataService } from 'src/app/services/data.service';
+import { Counter } from 'src/app/Utils/Utils';
+import { DataModelBase } from '../Base';
+import { InfrastructureJob } from '../infrastructureJob';
+import { DataModelCollectionBase } from './CollectionBase';
+
+export interface IInfrastructureCollectionItem {
+  Name: string;
+  Jobs: InfrastructureJob[];
+}
+
+export class InfrastructureCollectionItem extends DataModelBase<IInfrastructureCollectionItem> {
+  isThrottled: boolean = false;
+
+  executingMRJobs: InfrastructureJob[] = [];
+  allPendingMRJobs: InfrastructureJob[] = [];
+  completedMRJobs: InfrastructureJob[] = [];
+  throttledJobs: InfrastructureJob[] = [];
+
+  constructor(public data: DataService, public raw: IInfrastructureCollectionItem) {
+    super(data, raw);
+    this.updateInternal();
+  }
+
+  updateInternal(): Observable<any> {
+    this.isThrottled = this.raw.Jobs.some(job => job.raw.IsThrottled);
+    this.executingMRJobs = this.raw.Jobs.filter(job => job.raw.JobStatus === 'Executing' && Boolean(job.raw.IsActive))
+    this.allPendingMRJobs = this.raw.Jobs.filter(job => job.raw.JobStatus !== 'Completed' && !Boolean(job.raw.IsActive))
+    this.completedMRJobs = this.raw.Jobs.filter(job => job.raw.JobStatus === 'Completed');
+    this.throttledJobs = this.raw.Jobs.filter(job => job.raw.IsThrottled);
```

Comment:
I think we should add a banner when we get non empty Throttled Jobs saying the following job is getting throttled because there are other jobs in Executing State, and IS config is set to allow only certain number of Jobs at a time. @anantshankar17 , what do you think?